public/uw/start/index.html

```diff
@@ -0,0 +1,8 @@
+<!doctype html>
+ 
+<meta charset="utf-8">
+<meta http-equiv="refresh" content="0; url=https://backoffice.seattleflu.org/husky-musher/">
+ 
+<title>Husky Coronavirus Testing: Start</title>
+ 
+If you are not redirected automatically, <a href="https://backoffice.seattleflu.org/husky-musher/">continue on manually</a>.
```

Comment:
I hope that if participants end up seeing this page, they get a kick out of the "musher" naming 😆